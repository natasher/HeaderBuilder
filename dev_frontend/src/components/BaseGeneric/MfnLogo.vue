<template functional>
  <div class="logo"></div>
</template>

<script>
export default {

  name: 'MfnLogo',

}
</script>