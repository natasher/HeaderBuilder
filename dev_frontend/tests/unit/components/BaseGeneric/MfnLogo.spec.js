import { shallowMount } from '@vue/test-utils'
import MfnLogo from '../../../../src/components/BaseGeneric/MfnLogo'

describe('MfnLogo.vue', () => {

  test('should match snapshot', () => {
    const wrapper = shallowMount(MfnLogo)

    expect( wrapper.element ).toMatchSnapshot()
  })

})