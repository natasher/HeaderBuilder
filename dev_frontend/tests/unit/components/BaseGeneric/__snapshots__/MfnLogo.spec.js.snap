// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MfnLogo.vue should match snapshot 1`] = `
<div
  class="logo"
/>
`;
